server/views/pages/appointments/details/appointment.njk

```diff
@@ -0,0 +1,99 @@
+                    {
+                        key: {
+                            text: "Prisoner"
+                        },
+                        value: {
+                            html: appointmentPrisonerSummary(appointment.prisoners[0].firstName + ' ' + appointment.prisoners[0].lastName, appointment.prisoners[0].prisonerNumber, appointment.prisoners[0].cellLocation, dpsUrl)
+                        }
+                    },
+                    {
+                        key: {
+                            text: "Category"
+                        },
+                        value: {
+                            html: '<span data-qa="category">' + appointment.category.description + '</span>'
+                        }
+                    },
+                    {
+                        key: {
+                            text: "Location"
+                        },
+                        value: {
+                            html: '<span data-qa="location">' + appointment.internalLocation.userDescription + '</span>'
+                        }
+                    },
+                    {
+                        key: {
+                            text: "Date"
+                        },
+                        value: {
+                            html: '<span data-qa="start-date">' + appointment.startDate | formatDate('EEEE d MMMM yyyy') + '</span>'
+                        }
+                    },
+                    {
+                        key: {
+                            text: "Start time"
+                        },
+                        value: {
+                            html: '<span data-qa="start-time">' + appointment.startTime | formatDate('HH:mm') + '</span>'
+                        }
+                    },
+                    {
+                        key: {
+                            text: "End time"
+                        },
+                        value: {
+                            html: '<span data-qa="end-time">' + appointment.endTime | formatDate('HH:mm') + '</span>'
+                        }
+                    }
+                ],
+                classes: "govuk-!-margin-bottom-9"
+            }) }}
+
+            {% if appointment.createdBy %}
+                {% set createBy = appointment.createdBy | toTitleCase | initialiseName %}
+            {% else %}
+                {% set createBy = '[Unknown User]' %}
+            {% endif %}
+            {{ govukSummaryList({
+                rows: [
+                    {
+                        key: {
+                            text: "Created by"
+                        },
+                        value: {
+                            html: '<span data-qa="created-by">' + createBy + '</span>'
+                        }
+                    }
+                ],
+                classes: "govuk-!-margin-bottom-9"
+            }) }}
+
+            <div class="govuk-button-group govuk-!-display-none-print no-js-hidden">
```

Comment:
For context, I made this change because we're using a button for the print CTA on the unlock list page, and ideally we should follow a consistent pattern.

Generally unless the print CTA is navigating the user to a print friendly version of the page a button should be used because it can be confusing to use a link for non-navigational actions.

I've reverted this back to a link for now but I think this is something that's worth revisiting so we can agree on a pattern for print dialogs going-forward.